src/user_input_compiler.rs

```diff
@@ -0,0 +1,258 @@
+//! # user_input_compiler
+//!
+//! Parses Input with this Grammer after trimming the Input and ignoring Whitespaces:
+//! 
+//! Command         := "/" CommandWord
+//! CommandWord     := Status | Subscribe | Cancel | InvalidSyntax
```

Comment:
Strictly speaking, a CommandWord is actually more than just a word. What about renaming Command to FullCommand and CommandWord to Command?